azure-devops/run-tests.yml

```diff
@@ -6,19 +6,18 @@ parameters:
   type: string
 - name: targetArch
   type: string
-- name: ctestOptions
+- name: testTargets
   type: string
 - name: skipTesting
   type: boolean
 steps:
 - script: |
     call "%ProgramFiles%\Microsoft Visual Studio\2022\Preview\Common7\Tools\VsDevCmd.bat" ^
     -host_arch=${{ parameters.hostArch }} -arch=${{ parameters.targetArch }} -no_logo
-    ctest --verbose ${{ parameters.ctestOptions }}
+    cmake --build "$(buildOutputLocation)" --target ${{ parameters.testTargets }}

```

Comment:
Bug: this isn't going to run multiple test targets (as intended) if tests in the early targets fail. I think we need to run `ninja` directly so we can add `-k0` to get the desired behavior.